autoload/lsp/util.vim

```diff
@@ -133,6 +138,27 @@ export def LspFileToUri(fname: string): string
   return uri
 enddef
 
+# Convert POSIX paths as used in Cygwin to native Windows paths
+def CygwinToWindowsPath(path: string): string
+  if path =~? '^\/cygdrive\/'
+    # Convert paths of the form "/cygdrive/c/foo/bar" to "C:/foo/bar"
+    return substitute(path, '^\/cygdrive\/\(\a\)\/', '\=submatch(1) .. ":/"', "")
+  elseif path =~? '^\/'
+    # Convert paths of the form "/home/pete/foo" to "C:/cygwin64/home/pete/foo"
+    if len(g:cygwinroot) == 0
+      # https://stackoverflow.com/a/7449029/273348
+      g:cygwinroot = substitute(system("reg query HKEY_LOCAL_MACHINE\\\\SOFTWARE\\\\Cygwin\\\\setup /v rootdir | grep rootdir"),
```

Comment:
For readability, can you assign the command to a local variable and then use it in the system() call?